docs/references/kontena-yml.md

```diff
@@ -15,6 +15,124 @@ You can use environment variables in configuration values with a Bash-like ${VAR
 ## Service configuration reference
 > **Note:** Kontena supports both Docker Compose file versions respectively. Volumes and networks introduced in version 2 are not supported by Kontena yet. See more details about versioning on [Docker Compose documentation](https://docs.docker.com/compose/compose-file/#versioning)
```

Comment:
Could these "volumes and networks not supported" be moved under `#### Not supported in Kontena` heading -- I remember misinterpreting this somehow in the past.

Also, `working_dir:` is not supported in kontena.yml, that would fit under this category.